src/App.svelte

```diff
@@ -2,6 +2,14 @@
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
   console.log(exampleData);
```

Comment:
Remove unnecessary console log

Reply:
that one's not my fault, but I'll do it

Reply:
don't do it, let's not do this "while we're here" garbage

Reply:
that's operationally risky

Reply:
**its right there**

Reply:
but its free real estate